paper.bib

```diff
@@ -1,58 +1,38 @@
-@book{schroeder2004visualization,
-  title={The visualization toolkit: an object-oriented approach to 3D graphics},
-  author={Schroeder, Will J and Lorensen, Bill and Martin, Ken},
-  year={2004},
-  publisher={Kitware}
-}
-
-@book{sellers2016vulkan,
-  title={Vulkan programming guide: The official guide to learning vulkan},
-  author={Sellers, Graham and Kessenich, John},
-  year={2016},
-  publisher={Addison-Wesley Professional}
-}
-
-@article{harris2020array,
-  title={Array programming with NumPy},
-  author={Harris, Charles R and Millman, K Jarrod and van der Walt, St{\'e}fan J and Gommers, Ralf and Virtanen, Pauli and Cournapeau, David and Wieser, Eric and Taylor, Julian and Berg, Sebastian and Smith, Nathaniel J and others},
-  journal={Nature},
-  volume={585},
-  number={7825},
-  pages={357--362},
-  year={2020},
-  publisher={Nature Publishing Group}
+@article{abadi2016tensorflow,
+  title={Tensorflow: Large-scale machine learning on heterogeneous distributed systems},
+  author={Abadi, Mart{\'\i}n and Agarwal, Ashish and Barham, Paul and Brevdo, Eugene and Chen, Zhifeng and Citro, Craig and Corrado, Greg S and Davis, Andy and Dean, Jeffrey and Devin, Matthieu and others},
+  journal={arXiv preprint arXiv:1603.04467},
+  year={2016}
 }
 
-@article{virtanen2020scipy,
-  title={SciPy 1.0: fundamental algorithms for scientific computing in Python},
-  author={Virtanen, Pauli and Gommers, Ralf and Oliphant, Travis E and Haberland, Matt and Reddy, Tyler and Cournapeau, David and Burovski, Evgeni and Peterson, Pearu and Weckesser, Warren and Bright, Jonathan and others},
-  journal={Nature methods},
-  volume={17},
-  number={3},
-  pages={261--272},
-  year={2020},
-  publisher={Nature Publishing Group}
+@inproceedings{campagnola2015vispy,
+  title={VisPy: harnessing the GPU for fast, high-level visualization},
+  author={Campagnola, Luke and Klein, Almar and Larson, Eric and Rossant, Cyrille and Rougier, Nicolas P},
+  booktitle={Proceedings of the 14th Python in Science Conference},
+  year={2015},
+  doi={10.25080/majora-7b98e3ed-00e}
 }
 
-@inproceedings{mckinney2010data,
-  title={Data structures for statistical computing in python},
-  author={McKinney, Wes and others},
-  booktitle={Proceedings of the 9th Python in Science Conference},
-  volume={445},
-  pages={51--56},
-  year={2010},
-  organization={Austin, TX},
-  doi={10.25080/majora-92bf1922-00a}
+@article{coumans2013bullet,
+  author  = {Coumans, Erwin and others},
+  journal = {Open source: bulletphysics. org},
+  title   = {Bullet physics library},
+  year    = {2013},
+  number  = {49},
+  pages   = {5},
+  volume  = {15},
+  url     = {http://www.bulletphysics.org},
 }
```

Comment:
not sure if the spaces are ok or will generate an error

Reply:
Those spaces were added by a bibtex manager but I removed them for consistency purposes.